CHANGELOG

```diff
@@ -13,6 +13,10 @@ Qtile x.x.x, released xxxx-xx-xx:
           `group_name`. For the time being a log warning is in place and a
           migration is added. In the future `groupName` will fail.
         - Add `min/max_ratio` to Tile layout and fix bug where windows can extend offscreen.
+        - Add ability for widget `mouse_callbacks` to take `lazy` calls (similar to keybindings)
+        - Add `aliases` to `lazy.spawncmd()` which takes a dictionary mapping convenient aliases
+          to full command lines.
+        - Add a new 'prefix' option to the net widget to display speeds with a static unit (e.g. MB).
 
 Qtile 0.18.1, released 2021-09-16:
```

Comment:
Looks like something didn't quite go right when you rebased - there are some things that are appearing as added here which were already there.

Reply:
I must confess that git is quit new to me. I tried a `git rebase origin/master` and get
`Current branch Net_Widget_Option is up to date.`.

So the rebase did not fix the conflict. As a workaround i removed the change i made in the `CHANGELOG`. Currently I do not have the time to take a closer look at the `rebase` functionality. If you some easy commands for me to start with i may give it another try. Otherwise its fine for me to not mention it in the Changelog.

Reply:
No worries at all! Removing it from the changelog works - I can merge the main change and then add your changelog entry to the changelog in a follow-up commit :)

What would work to resolve the conflict is `git pull --rebase origin master` (or "upstream" if you have configured "upstream" to point to qtile/qtile, as "origin" will likely point to your own fork!). This will then update your local branch with your PR's commit removed initially, and then will try to put it back on that new base. It will conflict doing so, and prompt you to edit the conflicting files. The changelog would then have an annotation in the file indicating the conflict to make it easy to resolve. It can be pretty confusing for sure!